autorest/codegen/models/operation.py

```diff
@@ -132,8 +132,8 @@ def python_name(self) -> str:
     def request_content_type(self) -> str:
         return next(iter(
             [
-                cast(ConstantSchema, p.schema).constant_value for p in self.parameters.constant
-                if p.serialized_name == "content_type"
+                cast(ConstantSchema, p.schema).get_declaration(cast(ConstantSchema, p.schema).value)
```

Comment:
You don't need the first cast anymore (all schema have a get_declaration). Right?

Reply:
yep, removing!